src/common/types.hpp

```diff
@@ -1,16 +1,33 @@
 #pragma once
 
+#include <libdevcore/Address.h>
 #include <libdevcore/FixedHash.h>
-#include <libethereum/Transaction.h>
+#include <libdevcrypto/Common.h>
 
 #include <boost/asio.hpp>
 #include <boost/multiprecision/cpp_int.hpp>
 #include <chrono>
+#include <filesystem>
 #include <string>
 #include <type_traits>
 
 namespace taraxa {
 
+namespace fs = std::filesystem;
+
+using dev::Address;
+using dev::AddressSet;
+using dev::bytes;
+using dev::bytesConstRef;
+using dev::h256;
+using dev::h256Hash;
+using dev::h256s;
+using dev::h64;
+using dev::Secret;
+using dev::u256;
+
```

Comment:
Having using in headers is very bad practice. You just pollute namespaces thanks to this as every file that includes "types.hpp" will contain these usings, which is pretty bad. Here are some explanation related to this: https://jmmv.dev/2013/12/header-files-avoid-c-using-directives.html
https://stackoverflow.com/questions/19940081/using-keyword-in-header-implementation-in-c

Reply:
It is equivalent to a bunch of typedefs that we already have in that header. Therese particular types are so ubiquitous in our code that I thought it is okay to have them globally defined in our namespace. Would you be more accepting to `using h64 = dev::h64` for example (exactly same thing)? Or you think those type names should not belong to `taraxa` namespace at all?

Reply:
I think it should not be in taraxa namespace at all but I see that this practice is broadly used almost everywhere so it might be more of a task for bigger refactor...